vector/src/main/java/im/vector/app/features/home/room/list/RoomListSectionBuilderSpace.kt

```diff
@@ -431,72 +343,6 @@ class RoomListSectionBuilderSpace(
                            spaceFilterStrategy: RoomListViewModel.SpaceFilterStrategy = RoomListViewModel.SpaceFilterStrategy.NONE,
                            countRoomAsNotif: Boolean = false,
                            query: (RoomSummaryQueryParams.Builder) -> Unit) {
-<<<<<<< HEAD
-        withQueryParams(
-                { query.invoke(it) },
-                { roomQueryParams ->
-                    val name = stringProvider.getString(nameRes)
-                    session.getFilteredPagedRoomSummariesLive(
-                            roomQueryParams.process(spaceFilterStrategy, appStateHandler.safeActiveSpaceId()),
-                            pagedListConfig
-                    ).also {
-                        onUpdatable(it)
```

Comment:
Please restore this line which is specific to [tchap](https://github.com/tchapgouv/tchap-android-v2/pull/107/files#diff-352229f0f40a6d88ffa92f3595ae06250954e13d53e380c1668eb67d50252026R300) (and add a tchap comment) or delete it if it is not necessary anymore